### ISSUE TITLE
Error finding nativescript-barcodescanner

### ISSUE THREAD

**Issue by Edgesyntax-Kwame** [EddyVerbruggen/nativescript-barcodescanner#21]
I'm receiving an Unhandled Exception error after including the nativescript-barcodescanner plugin.
console shows that all plugins were prepared 
```
Successfully prepared plugin nativescript-barcodescanner for android.
Successfully prepared plugin tns-core-modules for android.
Successfully prepared plugin tns-core-modules-widgets for android.
Project successfully prepared
```

```
tns version 2.0.1
```

**Comment by EddyVerbruggen** [comment]
Can you also post the error please?

**Comment by Edgesyntax-Kwame** [comment]
Sure
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.nativescript.wmsmobile/com.tns.NativeScriptActivity}: com.tns.NativeScriptException: Failed to find module: "nativescript-barcodescanner", relative to: /app/tns_modules/
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2416)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
        at android.app.ActivityThread.-wrap11(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by: com.tns.NativeScriptException: Failed to find module: "nativescript-barcodescanner", relative to: /app/tns_modules/
        at com.tns.Module.resolvePathHelper(Module.java:220)
        at com.tns.Module.resolvePath(Module.java:60)
        at com.tns.Runtime.callJSMethodNative(Native Method)
        at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:861)
        at com.tns.Runtime.callJSMethodImpl(Runtime.java:726)
        at com.tns.Runtime.callJSMethod(Runtime.java:712)
        at com.tns.Runtime.callJSMethod(Runtime.java:693)
        at com.tns.Runtime.callJSMethod(Runtime.java:683)
        at com.tns.NativeScriptActivity.onCreate(NativeScriptActivity.java:13)
        at android.app.Activity.performCreate(Activity.java:6237)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
        ... 9 more
```

**Comment by EddyVerbruggen** [comment]
When do you see that error? When running `tns build android`, or at runtime? If the latter: can you post your project somewhere?

**Comment by EddyVerbruggen** [comment]
When do you see that error? When running `tns build android`, or at runtime? If the latter: can you post your project somewhere?

I'm asking because I just tried a simple starter app with this plugin and tns 2.0.1 on Android without issues.

**Comment by Edgesyntax-Kwame** [comment]
i'm seeing the error when running `tns livesync android --emulator --watch`

**Comment by EddyVerbruggen** [comment]
Have you already tried to do a `tns run android` to see if it's a problem with your code or livesync?

**Comment by Edgesyntax-Kwame** [comment]
Building the application using `tns run android` solved the problem.

I'm now getting a TypeError  message if no message option is provided.
but per the documentation a default message is assigned  if none is given.
`// Android only, default is 'Place a barcode inside the viewfinder rectangle to scan it.`

This is solved after providing a message.

```
JS: Error in barcodescanner.scan: TypeError: Cannot read property 'message' of undefined
JS: No scan: TypeError: Cannot read property 'message' of undefined
```

**Comment by EddyVerbruggen** [comment]
That doc is correct, but it was not expected you would not pass in any arguments. So `scan()` would fail indeed, but `scan({})` would have been fine.

I've just published 1.3.4 which allows `scan()` as well, like it already did on iOS.

Thanks for the report! 👍

**Comment by Edgesyntax-Kwame** [comment]
Thank you.
You've been a great help
